compiler/src/dmd/dsymbolsem.d

```diff
@@ -80,6 +80,8 @@ import dmd.visitor;
 
 enum LOG = false;
 
+private bool skipDeprecated;
```

Comment:
I don't like adding globals like this. I'll try to find a way to make this nicer. I can't add it to `DsymbolSemanticVisitor` because it's set by a call to `templateInstanceSemantic()` made from `dtemplate.d`.